doozerlib/runtime.py

```diff
@@ -851,14 +851,19 @@ def resolve_metadata(self):
         Allow http, https, ssh and ssh+git (all valid git clone URLs)
         """
 
-        if self.metadata_dir is None:
-            self.metadata_dir = constants.OCP_BUILD_DATA_RW
+        if self.data_path is None:
+            raise DoozerFatalError(
+                ("No metadata path provided. Must be set via one of:\n"
+                 "* data_path key in {}\n"
+                 "* doozer --datapath [PATH|URL]\n"
+                 "* Environment variable DOOZER_DATA_PATH\n"
```

Comment:
Yeah, i pointed out in an earlier comment about upper/lowercase vars. Here it's upper.

Reply:
That is an actual environment variable, the ones above are text replacement options for the foreach command. Very diff things.